pkg/kubelet/kubelet_pods.go

```diff
@@ -1673,11 +1644,11 @@ func getPhase(pod *v1.Pod, info []v1.ContainerStatus, podIsTerminal bool) v1.Pod
 	}
 
 	switch {
-	case pendingInitialization > 0 &&
+	case pendingRegularInitContainers > 0 || (pendingRestartableInitContainers > 0 &&
```

Comment:
nit: this code is hard to understand as the bracket is hard to notice at first. Perhaps only keep the `pendingRegularInitContainers > 0 ||` on this line and other two conditions to after the comment